Bump dependency version requirements

yorickps opened the pull request:
In order to be puppet 7.x compatible, a lot of module dependencies need
a bump in the upper limit version

treydock commented:
This won't work to pass tests until I do some updates to PCP module which this module depends on.

treydock commented:
@yorickps If you bump the max range for `pcp` module that should resolve the current test failure.  Also there is a bump for `geoip` that might be useful.

yorickps commented:
> @yorickps If you bump the max range for `pcp` module that should resolve the current test failure. Also there is a bump for `geoip` that might be useful.

Thanks for the info, both pcp and geoip have their upper limit version bumped.

treydock commented:
Can you please rebase against master branch or merge in latest master?  I added some fixes for changes made in PCP module.  To make those work I had to bump the PCP version in #25

yorickps commented:
> Can you please rebase against master branch or merge in latest master? I added some fixes for changes made in PCP module. To make those work I had to bump the PCP version in #25

Thanks, I saw the errors during the build. I fetched your upstream master and rebased against it.

treydock commented:
Looks like with the latest commit the update to geoip module might have been lost.

yorickps commented:
Ok, indeed, thanks. I've added geoip again

yorickps commented:
Ok, indeed, thanks. I've added geoip again.

treydock commented:
This was released with v7.5.0
